config/locales/pl.yml

```
@@ -193,6 +182,11 @@ pl:
         few: "%{count} minuty"
         many: "%{count} minut"
         other: "%{count} minut"
+      next_lesson: "Następna Lekcja"
```

YAMLlint (1.37.1)

[error] duplication of key "next_lesson" in mapping

(key-duplicates)